docs/en/interacting/monero-wallet-cli-reference.md

```diff
@@ -139,7 +139,7 @@ Wallet depends on a full node for all non-local operations. The following option
 
 | Option                         | Description
 |--------------------------------|--------------------------------------------------------------------------------------------------------------------------------------
-| `--subaddress-lookahead <arg>` | :warning: **Note: This flag can only be set during wallet creation or restore**<br><br>Accepts `m:n`, by default `50:200`. The first value is the number of accounts and the second value is the number of subaddresses per account. <br><br>The wallet will not check for payments to subaddresses further than `n` away from the last received payment. This can happen if you generated unique subaddresses for `n` clients in a row but none of them paid. <br><br>On the other hand the more subaddresses you set to look ahead, the longer it takes to create your wallet, because they must be pre-computed. This is normally not a concern, except for hardware wallets. On the Ledger the default value of `50:200` can take over 20 minutes (one time on wallet creation)!
+| `--subaddress-lookahead <arg>` | **Note: This flag can only be set during wallet creation or restore**<br>You may modify the lookahead while logged into the wallet by using<br>`set subaddress-lookahead m:n`<br>Accepts `m:n`, by default `50:200`. The first value is the number of accounts and the second value is the number of subaddresses per account. <br><br>The wallet will not check for payments to subaddresses further than `n` away from the last received payment. This can happen if you generated unique subaddresses for `n` clients in a row but none of them received a transfer.<br><br>On the other hand the more subaddresses you set to look ahead, the longer it takes to create your wallet, because they must be pre-computed. This is normally not a concern, except for hardware wallets. On the Ledger the default value of `50:200` can take over 20 minutes (one time on wallet creation)!
```

Comment:
currently `n-1` away

Reply:
Ikr lol. I debated whether to clarify that. Will change